### ISSUE TITLE
Styles are not applied with attribute binding

### ISSUE THREAD

**Issue by mannok** [nativescript-vue/nativescript-vue#557]
### Version
2.4.0

### Reproduction link
[https://play.nativescript.org/?template=play-vue&id=qXgtKz&v=2](https://play.nativescript.org/?template=play-vue&id=qXgtKz&v=2)
### Platform and OS info
Android 9 / NativeScript 6.1.2

### Steps to reproduce
1. check `bgRed` value in console by clicking  `get bgRed`    // you will get `false`
2. click toggle to invert `bgRed` from `false` to true
3. check `bgRed` value in console again by clicking `get bgRed`    // you will get `true`

### What is expected?
background color will change from blue to red after step 2

### What is actually happening?
background color is not changed

---
I can confirm this is not a bug of original vue framework and I have provided a working link of original vue framework as following.

https://codesandbox.io/embed/vue-template-qxwzn

**Comment by nuochong** [comment]
This seems to be a BUG with NativeScript. I have the same problem with nativescript-ng. The initialization background can produce the desired effect but dynamic modification is not effective.

**Comment by nuochong** [comment]
You can try this
```
<template>
  <Page class="page">
    <ActionBar title="Home" class="action-bar" />
    <ScrollView>
      <StackLayout class="home-panel" ref="homePanel" :class="{red:bgRed}">
        <Button @tap="bgRed = !bgRed">toggle</Button>
        <Button @tap="console.log(bgRed)">get bgRed</Button>
      </StackLayout>
    </ScrollView>
  </Page>
</template>

<script>
export default {
  data() {
    return {
      console,
      bgRed: false
    };
  }
};
</script>

<style scoped>
.home-panel {
  background-color: blue;
}
.red {
  background-color: red;
}
</style>
```

**Comment by mannok** [comment]
@nuochong Yes I know this will work. But how if I want to use dynamic attribute instead of dynamic class binding?

Is there any plan to fix this?

**Comment by tralves** [comment]
Yes, this is related to the way NS uses attributes. You have other ways to do that, though:

1. binding the attribute directly:
```xml
<StackLayout class="home-panel" ref="homePanel" :backgroundColor="bgRed?'red':'blue'">
```

or

```xml
<StackLayout class="home-panel" ref="homePanel" :backgroundColor="bgColor'">

computed: {
  bgColor() {
    return this.bgRed ? 'red':'blue';
  }
}
```

2. bind the `style` attribute:

```xml
<StackLayout class="home-panel" ref="homePanel" :style="stacklayoutStyle">

computed: {
  stacklayoutStyle() {
    return {
      'background-color': this.bgRed ? 'red':'blue',
    }
  }
}
```

**Comment by mannok** [comment]
> Yes, this is related to the way NS uses attributes. You have other ways to do that, though:
> 
> 1. binding the attribute directly:
> 
> ```
> <StackLayout class="home-panel" ref="homePanel" :backgroundColor="bgRed?'red':'blue'">
> ```
> 
> or
> 
> ```
> <StackLayout class="home-panel" ref="homePanel" :backgroundColor="bgColor'">
> 
> computed: {
>   bgColor() {
>     return this.bgRed ? 'red':'blue';
>   }
> }
> ```
> 
> 1. bind the `style` attribute:
> 
> ```
> <StackLayout class="home-panel" ref="homePanel" :style="stacklayoutStyle">
> 
> computed: {
>   stacklayoutStyle() {
>     return {
>       'background-color': this.bgRed ? 'red':'blue',
>     }
>   }
> }
> ```

Any way to keep custom attribute binding structure?

**Comment by mannok** [comment]
> This seems to be a BUG with NativeScript. I have the same problem with nativescript-ng. The initialization background can produce the desired effect but dynamic modification is not effective.

According to NativeScript respositry github. this seems to be a bug of NativeScript-vue. For anyone who are interested in. I have provided an issue link FYR.

[https://github.com/NativeScript/NativeScript/issues/7914#issuecomment-539359587](https://github.com/NativeScript/NativeScript/issues/7914#issuecomment-539359587)

**Comment by mannok** [comment]
> > This seems to be a BUG with NativeScript. I have the same problem with nativescript-ng. The initialization background can produce the desired effect but dynamic modification is not effective.
> 
> According to NativeScript respositry github. this seems to be a bug of NativeScript-vue. For anyone who are interested in. I have provided an issue link FYR.
> 
> [NativeScript/NativeScript#7914 (comment)](https://github.com/NativeScript/NativeScript/issues/7914#issuecomment-539359587)

According to what they say. Seems that notify should solve the updating event. However, It is not working with vue either. Reproduce link is provided.

[https://play.nativescript.org/?template=play-vue&id=qXgtKz&v=3](https://play.nativescript.org/?template=play-vue&id=qXgtKz&v=3)

**Comment by nuochong** [comment]
> > > This seems to be a BUG with NativeScript. I have the same problem with nativescript-ng. The initialization background can produce the desired effect but dynamic modification is not effective.
> > 
> > 
> > According to NativeScript respositry github. this seems to be a bug of NativeScript-vue. For anyone who are interested in. I have provided an issue link FYR.
> > [NativeScript/NativeScript#7914 (comment)](https://github.com/NativeScript/NativeScript/issues/7914#issuecomment-539359587)
> 
> According to what they say. Seems that notify should solve the updating event. However, It is not working with vue either. Reproduce link is provided.
> 
> https://play.nativescript.org/?template=play-vue&id=qXgtKz&v=3

@mannok The sample section of the code has an error with the property name

```
<template>
  <Page class="page">
    <ActionBar title="Home" class="action-bar" />
    <ScrollView>
      <StackLayout class="home-panel" ref="homepanel" :bgred="bgRed">
        <Button @tap="bgRed = !bgRed">toggle</Button>
        <Button @tap="console.log(bgRed)">get bgRed</Button>
        <Button @tap="forceUpdate">force update</Button>
      </StackLayout>
    </ScrollView>
  </Page>
</template>

<script>
export default {
  data() {
    return {
      console,
      bgRed: false
    };
  },
  methods: {
    forceUpdate() {
      const native = this.$refs.homepanel.nativeView;
      this.bgRed = !this.bgRed;
      native.notify({ eventName: 'bgredChange', object: native });
    }
  }
};
</script>

<style scoped>
.home-panel {
  background-color: blue;
}
.home-panel[bgred='true'] {
  background-color: red;
}
</style>
```

**Comment by mannok** [comment]
> > > > This seems to be a BUG with NativeScript. I have the same problem with nativescript-ng. The initialization background can produce the desired effect but dynamic modification is not effective.
> > > 
> > > 
> > > According to NativeScript respositry github. this seems to be a bug of NativeScript-vue. For anyone who are interested in. I have provided an issue link FYR.
> > > [NativeScript/NativeScript#7914 (comment)](https://github.com/NativeScript/NativeScript/issues/7914#issuecomment-539359587)
> > 
> > 
> > According to what they say. Seems that notify should solve the updating event. However, It is not working with vue either. Reproduce link is provided.
> > https://play.nativescript.org/?template=play-vue&id=qXgtKz&v=3
> 
> @mannok The sample section of the code has an error with the property name
> 
> ```
> <template>
>   <Page class="page">
>     <ActionBar title="Home" class="action-bar" />
>     <ScrollView>
>       <StackLayout class="home-panel" ref="homepanel" :bgred="bgRed">
>         <Button class="btn btn-primary" @tap="bgRed = !bgRed" text="toggle" />
>         <Button class="btn btn-primary" @tap="console.log(bgRed)" text="get bgRed" />
>         <Button class="btn btn-primary" @tap="forceUpdate" text="force update" />
>       </StackLayout>
>     </ScrollView>
>   </Page>
> </template>
> 
> <script>
> export default {
>   data() {
>     return {
>       console,
>       bgRed: false
>     };
>   },
>   methods: {
>     forceUpdate() {
>       const native = this.$refs.homepanel.nativeView;
>       this.bgRed = !this.bgRed;
>       native.notify({ eventName: 'bgredChange', object: native });
>     }
>   }
> };
> </script>
> 
> <style scoped>
> .home-panel {
>   background-color: blue;
> }
> .home-panel[bgred='true'] {
>   background-color: red;
> }
> </style>
> ```

@nuochong Thanks for pointing out. So I should be a solution by using notify. However, should it be fixed/enhanced? Since I think this is a power of vue framework and vue is supposed to handle these case
